### ISSUE TITLE
Following a user hides its About description

### ISSUE THREAD

**Issue by nephros** [poetaster/harbour-tooter#31]
Go to a users about page

expand the expando

read about description 

click follow

-> description vanishes

reopening the profile shows active follow button AND description again.

**Comment by poetaster** [comment]
Yes, annoying. If you can't remember the person/account in question, the description is not there to help :)

**Comment by poetaster** [comment]
Ok, it's not 100%. It seems to depend on what's in the persons bio area :) Might also be the latest update to the backend, since I can't reproduce this anymore. Can you?

**Comment by poetaster** [comment]
fixed in 1.3.0 (I believe).
